build.sbt

```diff
@@ -63,23 +63,22 @@ lazy val wartRemoverSettings = {
 }
 
 lazy val compileDeps = Seq(
-  "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "5.6.0",
-  "uk.gov.hmrc" %% "govuk-template"             % "5.61.0-play-27",
-  "uk.gov.hmrc" %% "play-partials"              % "7.1.0-play-27",
-  "uk.gov.hmrc" %% "play-ui"                    % "8.21.0-play-27", //needed for Optimizely
-  "uk.gov.hmrc" %% "agent-kenshoo-monitoring"   % "4.6.0-play-27",
+  "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.9.0",
+  "uk.gov.hmrc" %% "govuk-template"             % "5.70.0-play-28",
+  "uk.gov.hmrc" %% "play-partials"              % "8.2.0-play-28",
+  "uk.gov.hmrc" %% "play-ui"                    % "9.7.0-play-28", //needed for Optimizely
+  "uk.gov.hmrc" %% "agent-kenshoo-monitoring"   % "4.8.0-play-28",
   "uk.gov.hmrc" %% "agent-mtd-identifiers"      % "0.25.0-play-27",
-  "uk.gov.hmrc" %% "play-language"              % "4.12.0-play-27",
-  "uk.gov.hmrc" %% "play-frontend-hmrc"         % "0.45.0-play-27"
+  "uk.gov.hmrc" %% "play-language"              % "5.1.0-play-28",
```

Comment:
`play-language` gets pulled in transitively by `play-frontend-hmrc` as of version 0.68ish, so you can remove this :)

Good to see that bumping from v4 of `play-language` to v5 hasn't caused any issues

wondering if play-ui should still be there but best to leave it til we start with frontend library migrations I suppose... there is still bits of ui to tidy up and removing it should come under that work 👍

Reply:
Done! Removed both to no ill effect